src/components/recipe-form/schemas.ts

```diff
@@ -0,0 +1,30 @@
+import { z } from "zod";
+
+export const recipeSchema = z.object({
```

Comment:
As I said in previous comment:
maybe add avg price, time and difficulty level. And also will be good to have some kind of notes, which will be visable only if user will click on it. In notes people will write notes like:
"You can use tofu against of chicken, if you are a vegan" or something like that

Reply:
well notes fit into description field, no?